ansible/roles/common/tasks/packages.yml

```diff
@@ -1,10 +1,22 @@
 #vim: set ft=ansible:
 ---
 ######################### Install Basic Packages
+- name: Install DNF backend for Yum
```

Comment:
We should remove the use of backend dnf. Upstream is moving to the more generic "package" task.